### ISSUE TITLE
Table cell colors

### ISSUE THREAD

**Issue by amyjko** [amyjko/bookish#361]
## What's the problem?

Some authors would like to use colors on table cells, rows, and columns in order to highlight things.

## What's the design idea?

This is complex from a themeing perspective, if we want to keep things accessible and simple. For example, we could allow for arbitrary colors to be set on cell backgrounds, but then we'd need to allow text color to be set arbitrarily, to ensure readability. So we could define some highlight color in the theme (or reuse the one we have) and then allow cells to be highlighted. Or we could allow arbitrary colors, then compute a color from that color for the text.

The other challenge is that color information needs to be communicated some other way to screen readers.

## Who benefits?

Anyone who wants to compute information through color.

**Comment by orcmid** [comment]
In the past I have run across information about how to do that while preserving accessibility.  There are apparently pairs that tend to be best in case of color-blindness.  I'm certain there's advice about it somewhere.

**Comment by amyjko** [comment]
Yes, there's plenty of guidance on choosing colors; that's not the hard part. The hard part is designing a feature that communicates color information to people relying on screen readers; e.g.  making a cell red to communicate "badness", for example, not only has culturally situated meaning, but screen readers essentially ignore the color. There's also no good way of converting colors to descriptions of colors. So adding a feature like this opens the door to exclusion.
